### PR TITLE
display issues in websites details page (bug 1199620)

### PR DESCRIPTION
**Pull request by atiqueahmedziad**
r? @ngokevin

### CODE REVIEW

File: src/media/css/detail/header.styl [hunk 1]
```diff
@@ -11,6 +11,9 @@
     .mkt-app-button {
         margin: 0 auto;
     }
+    .app-notices {
+        border: 0 !important;
```
Comment (chuckharmston) [inline src/media/css/detail/header.styl]: Let's not use `!important`, even if that means having to reshuffle CSS elsewhere.

**Comment by ngokevin** [comment]
I bet you can probably just hide the whole app-notices section. They're websites so we shouldn't be mentioning platform compatibility. The platform data is just for API filtering.

**Comment by atiqueahmedziad** [comment]
Thanks kevin! I have hide the `app-notices` section. r? @ngokevin

**Comment by ngokevin** [comment]
looks great, thanks!

**Pull request merged**